Specs/Scene/ImageryLayerCollectionSpec.js

```diff
@@ -258,17 +258,52 @@ describe(
       expect(collection.isDestroyed()).toEqual(true);
     });
 
-    describe("pickImageryLayerFeatures", function () {
+    /**
+     * Repeatedly calls update until the load queue is empty.  Returns a promise that resolves
+     * once the load queue is empty.
+     *
+     * @param {Ray} ray The ray to test for intersection.
+     * @param {Scene} scene The scene.
+     * @return {Promise.<Boolean>}
+     *
+     * @private
+     *
+     */
+    function updateUntilDone(globe, scene) {
+      // update until the load queue is empty.
+      return pollToPromise(function () {
+        globe._surface._debug.enableDebugOutput = true;
+        scene.render();
+        return (
+          globe._surface.tileProvider.ready &&
+          globe._surface._tileLoadQueueHigh.length === 0 &&
+          globe._surface._tileLoadQueueMedium.length === 0 &&
+          globe._surface._tileLoadQueueLow.length === 0 &&
+          globe._surface._debug.tilesWaitingForChildren === 0
+        );
+      });
+    }
+
+    describe("pickImageryLayers", function () {
       var scene;
       var globe;
       var camera;
+      var customScene;
+      var customGlobe;
+      var customCamera;
 
       beforeAll(function () {
         scene = createScene();
         globe = scene.globe = new Globe();
         camera = scene.camera;
 
         scene.frameState.passes.render = true;
+        //NEW:
+        customScene = createScene();
+        customGlobe = customScene.globe = new Globe();
+        customCamera = customScene.camera;
+
+        customScene.frameState.passes.render = true;
```

Comment:
I originally added the set of custom variables because the spec `"pickImageryLayers returns undefined if no tiles are picked"` modifies `globe._surface._tilesToRender` and I do not want this change to impact the other specs.